README.rst

```diff
@@ -29,14 +29,24 @@ Demo
 To see a demo, use the test project included in the repo and perform the
 following steps (using a virtual environment is optional)::
 
-   git clone https://github.com/justinmayer/kagi
-   cd kagi
-   make install
-   make serve
-
-   cd testproj
-   python manage.py migrate
-   python manage.py createsuperuser
+Quick Set-up
+------------
+
+First, install Poetry_::
+
+   curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+Clone the kagi source code::
+
+   git clone https://github.com/justinmayer/kagi.git
+
+Then install the needed dependencies, set up the project and serve the demo::
+
+   poetry install
+   invoke setup
+   invoke serve
+
+Your local environment should now be ready to go!
```

Comment:
Perhaps add something like, "You should now be able to see the demo project login page in your browser at: http://localhost/kagi/login" ?